test/chunking-form/index.js

```diff
@@ -15,7 +15,7 @@ describe('chunking form', () => {
 
 		const config = loadConfig(samples + '/' + dir + '/_config.js');
 
-		if (config.skipIfWindows && process.platform === 'win32') return;
+		if ( !config || (config.skipIfWindows && process.platform === 'win32') ) return;
```

Comment:
The only worry now is that we are potentially mixing code styles. I know we stopped the spacing convention with the TS conversion, but if we want to maintain it for the JS and tests that's fine, but we should at least be consistent I think?

Reply:
Actually I do not really care about one formatting or the other. Formatting can easily be changed on way or another instantly via software and I think that is ultimately the way we should handle it to no longer waste time with this. If #1896 is advanced, it should be easy to let it cover some of the .js files as well.

Reply:
I will leave the formatting as it is to keep the diff smaller. No use trying to copy the arbitrary style of the rest of the file IMO though.